analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/Tests/VerifierTest.cs

```diff
@@ -159,6 +159,47 @@ public class Sample
         public void Verify_Cshtml() =>
             DummyWithLocationMapping.AddPaths("Dummy.cshtml").Verify();
 
+        [TestMethod]
+        public void Verify_Razor_WithFramework()
+        {
+            var verifierBuilder = DummyWithLocationMapping.AddPaths("Dummy.razor");
+            verifierBuilder.WithFramework("net6.0")
+                           .Invoking(x => x.Verify())
+                           .Should()
+                           .NotThrow();
```

Comment:
I wonder if we could go further and check if the actual csproj has the right TargetFramework but that would require parsing it, again...

Reply:
I think it could be done, the problem is that I'm not sure where to locate the .csproj file from here, any suggestions?